### PR TITLE
Add stringify method

### LINKED ISSUE #3: Impliment stringify metod
Example
```
var bemImport = require('bem-import');

var str = 'b:block m:theme=normal';
var cells = bemImport.parse(str);
bemImport.stringify(cells);
// b:block m:theme=normal
```

### PR DESCRIPTION
**Pull request by Yeti-or**
fix #3

### CODE REVIEW

**Comment by Yeti-or** [comment]
@zxqfox I didn't implement throwing on different blocks in array, or block + elem.
Are you sure that we need validation of input ?

File: tests/stringify.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            { block : 'popup', mod : { name : 'autoclosable' } }
+        ])).to.be.eql('b:popup m:autoclosable');
+    });
+
+    it('should stringify block with modifier', () => {
+        expect(s([
+            { block : 'popup' },
+            { block : 'popup', mod : { name : 'autoclosable' } },
+            { block : 'popup', mod : { name : 'autoclosable', val : 'yes' } }
+        ])).to.eql('b:popup m:autoclosable=yes');
```
Comment (awinogradov) [inline tests/stringify.js]: eql is the deep method

Comment (Yeti-or) [reply tests/stringify.js]: fixed

File: index.js [hunk 2]
```diff
@@ -60,6 +59,41 @@ function parse(importString, ctx) {
...
+        return acc;
+    }, { m : {} });
+
+    return ['b', 'e', 'm', 't'].reduce((str, k) => str += tmpl[k](merged[k]), '');
+}
+
+const tmpl = {
+    b : b => `b:${b}`,
+    e : e => e ? ` e:${e}` : '',
+    m : m => Object.keys(m).reduce((acc, name) => acc + `${tmpl.mn(name)}${tmpl.mv(m[name])}`, ''),
```
Comment (qfox) [inline index.js]: Давай `Object.keys(m).map(n => tmpl.mn(n) + tmpl.mv(m[n])).join('');`?

File: index.js [hunk 3]
```diff
@@ -60,6 +59,41 @@ function parse(importString, ctx) {
...
+ * ```js
+ * stringify([{ block : 'button' }, { block : 'button', mod : { name : 'theme', val : 'normal' } }])
+ * // 'b:button m:theme=normal'
+ * ```
+ * @public
+ * @param {BemCell[]} - Array of bem-entites to merge into bem-import string notation
+ * @returns {String}
+ */
+function stringify(cells) {
+    const merged = [].concat(cells).reduce((acc, cell) => {
```
Comment (zxqfox) [inline index.js]: assertov dobavim?

File: package.json [hunk 4]
```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "pretest": "npm run lint",
     "test": "npm run specs",
-    "specs": "mocha",
-    "cover": "nyc mocha",
+    "specs": "mocha tests",
+    "cover": "nyc mocha tests",
```
Comment (qfox) [inline package.json]: ну не делай так, test же стандарт

**Review by qfox — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -84,6 +85,15 @@ Note that, using context exludes `{ block: 'button'}` from result.
 
 So `parse('m:theme=normal', { block: 'button' })` is not same as `parse('b:button m:theme=normal')`
 
+### stringify
+
+Parameter | Type     | Description
+----------|----------|------------------------------------------------------------------------------
+`entities`| `array`  | Array of [BEM entities] to merge into bem-import string [notation](#notation)
```
Comment (qfox) [inline README.md]: into import string notation

File: README.md [hunk 6]
```diff
@@ -84,6 +85,15 @@ Note that, using context exludes `{ block: 'button'}` from result.
...
+
+Forms a string from [bem entities]. Be aware to merge only one type of entities.
```
Comment (qfox) [inline README.md]: BEM entities

File: README.md [hunk 7]
```diff
@@ -84,6 +85,15 @@ Note that, using context exludes `{ block: 'button'}` from result.
...
+Array must contain block and it's mods, or elem and it's mods.
```
Comment (qfox) [inline README.md]: modifiers

The array should contains one block or one elem and optionally it's modifiers.

File: index.js [hunk 8]
```diff
@@ -60,6 +59,41 @@ function parse(importString, ctx) {
...
+/**
+ * Get bem-import string notation representation of a bem-cells array.
+ *
+ * Example:
+ * ```js
+ * stringify([{ block : 'button' }, { block : 'button', mod : { name : 'theme', val : 'normal' } }])
+ * // 'b:button m:theme=normal'
+ * ```
+ * @public
+ * @param {BemCell[]} - Array of bem-entites to merge into bem-import string notation
```
Comment (qfox) [inline index.js]: BEM entities

Comment (qfox) [reply index.js]: import string notation

File: index.js [hunk 9]
```diff
@@ -60,6 +59,41 @@ function parse(importString, ctx) {
     }, []);
 }
 
+/**
+ * Get bem-import string notation representation of a bem-cells array.
```
Comment (qfox) [inline index.js]: Creates BEM import string notation of passed array with BemCell objects. ?

File: index.js (hunk 8 above)
Comment (qfox) [reply index.js]: Array → Set?

**Review by qfox — COMMENTED** [review]
Po kodu vrode norm

**Pull request merged**